lib/validators.js

```diff
@@ -188,8 +191,8 @@ const Validators = ({
         401: 'Unauthorized',
       };
 
-      if (!(cacheableStatusCodes[response.statusCode])) {
-        evaluator.flagStorable(false);
+      if ((cacheableStatusCodes[response.statusCode])) {
```

Comment:
why are you changing the evaluation logic? with this change you explicitly flag the evaluator as storable just because the response statuscode matches... but there might be other evaluators that find the response as not storable...
IMHO this change defeats the purpose of the evaluator

Reply:
If any of the previous evaluators mark response as "not-storable", the statusCode evaluator will never be reached. But if all previous evaluators have returned "false", and statusCode = 200, this evaluator will also return false (in the current version). 

We end up with having "storable" prop as undefined, even though the response might be valid for storing.

btw, response evaluator is not assumed storable by default, maybe we could make it? But then, there is no need for "e-tag" and "last-modified" validators, since they mark response as storable already.